game/src/js/objects/Character.js

```diff
@@ -90,7 +90,7 @@ export default class Character extends Phaser.Physics.Arcade.Sprite {
       this.setVelocity(vector.x * this.props.baseSpeed, vector.y * this.props.baseSpeed);
     }
     let animation = 'stance';
-    if(vector.length() != 0) {
+    if(vector.length != 0) {
```

Comment:
I think length is not a property of vector, but instead a method as it was originally. Could you please change this back?

Reply:
It crashes the game the way it current was.  I don't think I can get it to work without changing it.

Reply:
According to the docs, this should be a method: https://photonstorm.github.io/phaser3-docs/Phaser.Math.Vector2.html#length.